### DISCUSSION TITLE
[Feature Request] Add the option to enable a 2nd icon (and separate config) for Sonarr and Radarr

### DISCUSSION THREAD

**Discussion by Zevell** [Purfview/IMDb-Scout-Mod#221] [Ideas (implemented)]
### The Idea
Essentially the idea is to add the option to enable another icon for Sonarr, and/or Radarr, if the user enables it. This would allow users such as myself who use multiple instances of Radarr to easily add movies to the instance I want.

### Why
The use-case for this is that some people may prefer having a 2nd instance of Radarr specifically dedicated to 4K movies, as they'd have different quality definitions and potentially other settings for 4K movies.

In the case of Sonarr, most commonly a 2nd instance is used for anime, as anime benefits from vastly different quality settings (file sizes can be much smaller, while having great quality). 

### Implementation and Visual
Where should these icons show up? In the same bar that the current Radarr and Sonarr icons appear, right next to them ideally. A tooltip will show when hovering to indicate which instance of Radarr/Sonarr it is. Ideally, the user can change the tooltip text.

A visual of what I am talking about (my mouse cursor is invisible, but I am hovering over the icons in these pictures):
![firefox_w2m2Lry1Ye](https://github.com/Purfview/IMDb-Scout-Mod/assets/29517448/bce5d1bd-1ea4-424d-af29-448c6d3d7c3d)
![firefox_iWIY5vYAk7](https://github.com/Purfview/IMDb-Scout-Mod/assets/29517448/fd3b1f43-d85d-458f-89bc-e5fbcc3eadf5)

**Comment by Purfview** [comment]
You can open a new PR to implement this.

EDIT: Implementation PR -> https://github.com/Purfview/IMDb-Scout-Mod/pull/222

**Reply by Zevell** [reply]
Are you able to check out my fork and help me out? I have made decent progress on adding the 2nd instance of Radarr, but unfortunately I am encountering an issue with the icons being broken in the settings menu. I cannot figure out why. If you run my version of the script you'll see the error "TypeError: custom_sites[index] is undefined". Sorry for messaging about it here, not really sure how else to ask you for help.

**Reply by Purfview** [reply]
Maybe it's something that you didn't added new 24th section to GM_config.init.
Search and look at `#imdb_scout_section_`.

**Reply by Zevell** [reply]
Thanks that appears to have been it, I had to increment the lines that contained "$('#imdb_scout').contents().find('#imdb_scout_section_11')" by 1. So 11 became 12, etc. I am not entirely sure why it fixed it, once I make the pull request, I'll mention it so you can look over it extra carefully to make sure I didn't accidentally break something. :)
